Editor/CKEditor/Resources/views/editorheader.html.twig

```diff
@@ -1,16 +1,21 @@
 <!-- start Scribite with CKEditor for {{ modname }} -->
 {# needs to be placed before the CKEditor files are included. #}
-{#<script type="text/javascript">#}
+{# <script type="text/javascript"> #}
     {# I'm not sure if a relative path or full path with schemeAndHost is better here #}
-    {#var CKEDITOR_BASEPATH = '{{ app.request.schemeAndHttpHost }}{{ asset('editors/ckeditor/ckeditor/') }}';#}
-{#</script>#}
+    {# var CKEDITOR_BASEPATH = '{{ app.request.schemeAndHttpHost }}/modules/Scribite/Editor/CKEditor/vendor/ckeditor/'; #}
+{# </script> #}
 {{ pageAddAsset('javascript', asset('editors/ckeditor/js/CKEditor.url.js'), 99) }}
 {{ pageAddAsset('javascript', asset('editors/ckeditor/ckeditor/ckeditor.js'), 100) }}
 {{ pageAddAsset('javascript', asset('editors/ckeditor/ckeditor/adapters/jquery.js'), 101) }}
 {{ pageAddAsset('javascript', asset('editors/ckeditor/js/CKEditor.ajaxApi.js'), 102) }}
 {{ pageAddAsset('stylesheet', asset('editors/ckeditor/css/style.css')) }}
-{% set useckfinder, usekcfinder = false, false %}
-{% if editorVars.filemanagerpath~'/ckfinder.html'|php('file_exists') %}
+{% set usezfiler, useckfinder, usekcfinder = false, false, false %}
+{% if 'modules/Zfiler/javascript/zfiler_cke.js'|php('file_exists') %}
+    {% set usezfiler = true %}
+    <link rel="stylesheet" href="web/jquery-ui/themes/base/jquery-ui.css" type="text/css">
+    <script type="text/javascript" src="modules/Zfiler/javascript/zfiler_cke.js"></script>
```

Comment:
Looks much like legacy. Should use `zasset` function instead.

Reply:
Definitely. Anyway moving to ```zasset``` needs some testing, I am not familiar with the way and order it loads scripts.